### ISSUE TITLE
Went to run 210-azure-portworx and got the following error

### ISSUE THREAD

**Issue by mjperrins** [cloud-native-toolkit/terraform-azure-portworx#8]
Went to run 210-azure-portworx and got the following error , this was generated as part of the Maximo automation

```
drwxr-xr-x    3 root     root          4096 May 22 14:35 .
drwxr-xr-x    8 root     root          4096 May 22 14:35 ..
-rw-r--r--    1 root     root           532 May 22 14:35 210-azure-portworx-storage.auto.tfvars
drwxr-xr-x    2 root     root          4096 May 22 14:35 docs
-rw-r--r--    1 root     root          1288 May 22 14:35 main.tf
-rw-r--r--    1 root     root            23 May 22 14:35 providers.tf
lrwxrwxrwx    1 root     root            36 May 22 14:35 terraform.tfvars -> /workspaces/current/terraform.tfvars
-rw-r--r--    1 root     root          2726 May 22 14:35 variables.tf
-rw-r--r--    1 root     root            96 May 22 14:35 version.tf
bash-5.1# terraform init
There are some problems with the configuration, described below.

The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│ Error: Argument or block definition required
│
│ On main.tf line 18: An argument or block definition is required here.
```

**Comment by rich-ehrhardt** [comment]
Same as issue #9 - cluster_name variable needs to be in quotes to avoid error when running iascable.

**Comment by rich-ehrhardt** [comment]
Resolved from issue #9
